apps/condo/domains/acquiring/schema/Recipient.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Generated by `createschema acquiring.Recipient 'organization:Relationship:Organization:CASCADE; tin:Text; iec:Text; bic:Text; bankAccount:Text; bankName?:Text; offsettingAccount?:Text; territoryCode?:Text; purpose?Text; name?:Text; isApproved:Checkbox; meta?:Json;'`
+ */
+
+const get = require('lodash/get')
+
+const { Text, Checkbox } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const access = require('@condo/domains/acquiring/access/Recipient')
+const { DV_FIELD, SENDER_FIELD, IMPORT_ID_FIELD } = require('@condo/domains/common/schema/fields')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { hasDvAndSenderFields } = require('@condo/domains/common/utils/validation.utils')
+const { DV_UNKNOWN_VERSION_ERROR } = require('@condo/domains/common/constants/errors')
+
+const { validateTin } = require('@condo/domains/acquiring/utils/validate/tin.utils')
+const { validateIec } = require('@condo/domains/acquiring/utils/validate/iec.utils')
+const { validateBic } = require('@condo/domains/acquiring/utils/validate/bic.utils')
```

Comment:
You can group all validation utils to one index file

Reply:
I don't think you should group them. Because in the future it is possible to expand validators for different countries